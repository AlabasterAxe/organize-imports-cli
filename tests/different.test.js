const runCli = require("./run-cli");

describe("--list-different", () => {
  runCli("cli/different", ["--list-different", "file1.js", "file2.js"]).test();
});
