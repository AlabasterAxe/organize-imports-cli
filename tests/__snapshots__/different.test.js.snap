// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--list-different (status) 1`] = `2`;

exports[`--list-different (stderr) 1`] = `""`;

exports[`--list-different (stdout) 1`] = `
"<dir>/file1.js
"
`;

exports[`--list-different (write) 1`] = `Array []`;
